modin/core/io/text/custom_text_dispatcher.py

```diff
@@ -0,0 +1,101 @@
+# Licensed to Modin Development Team under one or more contributor license agreements.
+# See the NOTICE file distributed with this work for additional information regarding
+# copyright ownership.  The Modin Development Team licenses this file to you under the
+# Apache License, Version 2.0 (the "License"); you may not use this file except in
+# compliance with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+"""Module houses `CustomTextExperimentalDispatcher` class, that is used for reading custom text files."""
+
+import pandas
+
+from modin.core.io.file_dispatcher import OpenFile
+from modin.core.io.text.text_file_dispatcher import TextFileDispatcher
+from modin.config import NPartitions
+
+
+class CustomTextExperimentalDispatcher(TextFileDispatcher):
```

Comment:
I am in favor of ExperimentalCustomTextDispatcher. The use of Experimental in the middle of a name looks weird to me.